--- origsrc/portaudio/configure.in	2007-11-13 15:22:56.000000000 -0600
+++ src/portaudio/configure.in	2008-05-30 00:20:00.000000000 -0500
@@ -229,7 +229,7 @@
   cygwin* )
 	dnl Cygwin configuration
 
-	OTHER_OBJS="src/hostapi/wmme/pa_win_wmme.o src/os/win/pa_win_hostapis.o src/os/win/pa_win_util.o";
+	OTHER_OBJS="src/hostapi/wmme/pa_win_wmme.o src/os/win/pa_win_hostapis.o src/os/win/pa_win_util.o src/os/win/pa_win_waveformat.o";
 	CFLAGS="$CFLAGS -DPA_NO_DS -DPA_NO_WDMKS -DPA_NO_ASIO -DPA_NO_WASAPI"
 	LIBS="-lwinmm -lm";
 	PADLL="portaudio.dll";
